src/main/java/frc/trigon/robot/misc/ShootingCalculations.java

```diff
@@ -137,7 +137,7 @@ private TargetShootingState calculateTargetShootingState(TargetShootingState sta
     private TargetShootingState calculateTargetShootingState(Translation3d shootingVector) {
         final MirrorableRotation2d targetRobotAngle = new MirrorableRotation2d(getYaw(shootingVector), false);
         final Rotation2d targetPitch = getPitch(shootingVector);
-        final double targetVelocity = tangentialVelocityToAngularVelocity(shootingVector.getNorm());
+        final double targetVelocity = tangentialVelocityToAngularVelocity(shootingVector.getNorm()) + ShootingConstants.SPEAKER_SHOT_STANDING_EXTRA_VELOCITY_BUFFER_ROTATIONS_PER_SECOND.get();
```

Comment:
Clean this up. This is something that should be calibrated and used in every season with shooting